app/configurator/components/chart-configurator.tsx

```diff
@@ -61,20 +102,78 @@ export const ChartConfigurator = ({
   state: ConfiguratorStateConfiguringChart;
 }) => {
   const locale = useLocale();
-  const [{ data }] = useDataCubeMetadataWithComponentValuesQuery({
-    variables: {
+  const [, dispatch] = useConfiguratorState();
+  const variables = React.useMemo(
+    () => ({
+      date: new Date(),
```

Comment:
Why do you have a `date` here?

Reply:
Added some docs about this in a new commit, this is to force urql to fetch with the new filters. Otherwise urql does not check the order of the filters.

Reply:
Great, thanks!